apps/data/lib/data/opportunities.ex

```diff
@@ -6,7 +6,11 @@ defmodule Data.Opportunities do
   alias Data.{Opportunity, Repo}
 
   def all, do: Repo.all(Opportunity)
-
+  def all_with_project do
```

Comment:
Add an empty line between different functions, above and below this one.

We probably don't need a new method here, this use-case is the only need (for now) for the `all/0` function.  Just delete the function above and rename this one, we can improve it later as part of other work.